src/main/java/com/antelif/library/application/error/GenericError.java

```diff
@@ -36,6 +36,8 @@ public enum GenericError {
   CANNOT_CANCEL_FINALIZED_TRANSACTION(33, "Cannot cancel finalized transaction."),
   CANNOT_CANCEL_PARTIALLY_UPDATED_TRANSACTION(
       34, "Cannot cancel transaction that has books returned. Return all books to finalize."),
+  DUPLICATE_BOOKS_IN_TRANSACTION(
```

Comment:
Fix text here to `Transaction cannot contain more than one books with the same ISBN.`